lib/index.js

```diff
@@ -62,10 +69,14 @@ export {
   // narrative components
   NarrativeItineraries,
   NarrativeItinerary,
+  NarrativeProfileOptions,
 
   // actions
+  findNearbyStops,
+  getCurrentPosition,
   setAutoPlan,
-  setShowExtendedSettings,
+  setLocationToCurrent,
+  setMapCenter,
```

Comment:
interesting that set `setQueryParam` isn't exported.  Would be interested to hear your thoughts on why.